TSG/LCM/Add-Node-Failed-at-SetTrustedHostsOnNewNode-Step.md

```diff
@@ -0,0 +1,164 @@
+# How to Resume Add Node operation on BrownField Environment failed at SetTrustedHostsOnNewNode Step
+# Description 
+
+This article describes how to resume Add node operation on a brownfield upgraded cluster when failed at SetTrustedHostsOnNewNode.
+# Symptoms
+ScaleOutOperation action plan for Add node operation will fail with below message at SetTrustedHostsOnNewNode step.
+```
```

Comment:
I think we should also clarify that this needs to be an environment that is upgraded from 22H2.